tests/cli/test_compile.py

```diff
@@ -1,21 +1,90 @@
 import itertools
+import os
 from typing import List
+from unittest import mock
 
 import pytest
 from typer.testing import CliRunner
 
 from amora.cli import app
+from amora.config import settings
 from amora.utils import clean_compiled_files, list_target_files
 
 runner = CliRunner(mix_stderr=False)
 
 
 def setup_function(_module):
     clean_compiled_files()
+    if os.path.exists(settings.manifest_path):
```

Comment:
```suggestion
    settings.manifest_path.unlink(missing_ok=True)
``` 

https://docs.python.org/3.8/library/pathlib.html#pathlib.Path.unlink